src/main/java/ca/corefacility/bioinformatics/irida/model/user/group/UserGroup.java

```diff
@@ -50,7 +52,7 @@
 	@Temporal(TemporalType.TIMESTAMP)
 	private Date modifiedDate;
 
-	@OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.REMOVE, mappedBy = "group")
+	@OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.REMOVE, mappedBy = "group")
```

Comment:
I'm a little hesitant about changing this to `EAGER` instead of `LAZY`. I wonder what impact it would have in other areas of the code. I guess if everything accessing `UserGroup` needs to load the `Users` then `EAGER` is appropriate. But maybe they aren't actually since you just added the `getUsers` method.

Reply:
I can, I just have to change the logic within ProjectServiceImpl.removeUserGroupFromProject and ProjectServiceImpl.addUserGroupToProject. I'm just struggling with how to do it.

Reply:
@deepsidhu85 your a little more familiar then me about lazy initialized collections, any comments.

Reply:
My understanding of Eager vs Lazy loading for collections is if the data needs to be available when executing the code then we should be using Eager. But if the data is required later and not at the time of code execution then to use Lazy loading. I don't have any issues with changing it as long as long as previous code using the usergroup model isn't broken. So we should test it out to make sure

Reply:
@deepsidhu85  previously no code that used UserGroup accessed Users, hence the lazy loading. With this new addition there are a couple instances that use the Users. I was more asking how you dealt with lazy initialized collections when you needed to access them.

Reply:
@ericenns Oh ok sorry about that. I can't recall if I've ever done that myself before but can take a look and get back to you